crates/cli/src/native_query/aggregation-operator-progress.md

```diff
@@ -0,0 +1,280 @@
+Arithmetic Expression Operators
+
+- [x] $abs - Returns the absolute value of a number.
```

Comment:
I assume that Mongo has no deep concept of introspection for these functions, so it's just a case of special casing each one we want to support?

Reply:
Unfortunately MongoDB doesn't have much of a concept of typing at all. I don't think there is any way to introspect for signatures for functions like these.

It's possible we could build our own function signature language, and write signatures ourselves. But I thought it would be easier to hard-code this stuff, especially since the query language has a closed, limited set of functions.

Reply:
I think this approach is absolutely fine, it's very clear what is going on which is great.